Patterns/Japanese/Japanese-Choice.yaml

```diff
@@ -0,0 +1,9 @@
+﻿---
+LangMarker: Eng
+#Boolean Recognizers
```

Comment:
This file was not updated with the Japanese values used to generate the ChoiceDefinitions.cs file. Please update them here.
Also, please add the generated definitions file in Typescript.

Reply:
Got it.